plutus-core/plutus-core/src/PlutusCore/Parser.hs

```diff
@@ -69,7 +69,7 @@ errorTerm = inParens $ Error <$> wordPos "error" <*> pType
 
 -- | Parser for all PLC terms.
 term :: Parser PTerm
-term = choice $ map try
+term = whitespace >> (choice $ map try
```

Comment:
I'd write it as

```haskell
term = do
    whitespace
    choice $ map try
        <...>
```

but feel free to ignore my nitpicking.

Reply:
I like `do` in general too actually.